types/googlemaps/index.d.ts

```diff
@@ -3718,9 +3718,13 @@ declare namespace google.maps {
         interface PlaceReview {
             aspects: PlaceAspectRating[];
             author_name: string;
-            author_url: string;
+            author_url?: string;
             language: string;
+            profile_photo_url: string;
+            rating: number;
```

Comment:
There's no mention of a `rating` field in the docs. If it exists, it's probably not an officially supported field and shouldn't be part of the typings.

Reply:
Oops, you're right-- it's not mentioned in these docs.
It is shown here: https://developers.google.com/places/web-service/details#PlaceDetailsResults, and it is returned by the service. Since this is the field that motivated my PR, is the proper thing to:
* Leave it out
* Make it optional
* Leave it in and add a ref to this doc
* Something else?

Thanks

Reply:
I have also submitted an issue to Google to request this addition (https://issuetracker.google.com/issues/156446921). I don't know that they'll treat it with a lot of urgency, though :).

Reply:
Which of the above would you prefer? I don't know what's standard for this kind of exercise. I will resolve the merge conflict once we reach a decision about how to proceed. Thanks.

Reply:
I'm not entirely sure either but I'd say it's fine with your extra doc link. @demensky what do you think?

BTW the definitions were split into multiple files so you have to merge master and move your definition changes into `reference/places-service.d.ts`:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/90c86db5b717b3687f090602ecb466347e8f50c2/types/googlemaps/reference/places-service.d.ts#L197-L204

Reply:
Please provide an example of code that will allow you to get an object with the fields described in the interface. This will greatly simplify the solution to the problem.

`rating` is not like a private field. I propose to add `// TODO: …` in which to describe the reason and links.